### ISSUE TITLE
Prophet predictor : bug or personnal mistake ?

### ISSUE THREAD

**Issue by NicholasJallan** [awslabs/gluon-ts#664]
Hi there,

I am trying to compare different algorithms for TS i have (frequency=1 value per day over the last 5 years).
As i first used directly Prophet to make some predictions, i was expecting to get the same results from GluonTS with the Prophet wrapper predictor. 

Basically, here the code i tried (as it's a sample from notebook, some imports might be missing): 

Creating all my dataframe with 5 years of data. Finally i have :
`
df.info()
`
<class 'pandas.core.frame.DataFrame'>
DatetimeIndex: 2130 entries, 2014-01-01 to 2019-10-31
Data columns (total 1 columns):
nb_sales    2130 non-null uint16
dtypes: uint16(1)
memory usage: 20.8 KB

The index of the dataframe is Timestamp.

Then i make a simple split to get train period and validation set :
`
train_data = df[df.index < '2019-09-01']
test_data = df[df.index >= '2019-09-01']
`
test_data contains here now 61 values (the 2 last months of my dataset).
There i start to get a prediction for the 2 last months, with the expectation it will be matching the test set.

`
from gluonts.model.prophet import ProphetPredictor
prophet = ProphetPredictor(freq='1D', prediction_length=60)

from gluonts.dataset.common import ListDataset

training_data = ListDataset(
    [{
        "start": train_data.index[0], 
        "target": train_data['nombre_vente'].values
    }],
    freq = "1D"
)

print(training_data)

for entry in training_data:
    print(entry)
    print(entry['target'])
    print (len(entry["target"]))
`

This print out :
<gluonts.dataset.common.ListDataset object at 0x7f166ba63290>
{'start': Timestamp('2014-01-01 00:00:00', freq='D'), 'target': array([ 389.,  853., 2258., ..., 1554., 1632.,  868.], dtype=float32), 'source': SourceContext(source='list_data', row=1)}
[ 389.  853. 2258. ... 1554. 1632.  868.]
2069

Then i try to get the forecast :
`
for forecast in prophet.predict(dataset=training_data):
    print(forecast)
    forecast.plot(color='g', prediction_intervals=[50.0, 90.0])
plt.grid(which='both')
`
Wich prints
gluonts.model.forecast.SampleForecast(freq="1D", info=None, item_id=None, samples=numpy.array([[...],[...],[... , -52.71077557165853, 1851.6787772382265, 1731.6565198466628, 2376.988059053198, 1422.4524262580624, 2064.955094437749, 585.4025024689103, 184.27866528567733, 2021.475500168474, 1913.9258219772787, 1911.1257732221304]], numpy.dtype("float64")), start_date=pandas.Timestamp("2019-09-01 00:00:00", freq="D"))

For obvious reasons of length, i replaced values by [...], but there is a long list actual values standing there.
Finally, this code fails during the plot method at the line 206 of gluonts/model/forecast.py : 
`
--> 206         p50_series.plot(color=color, ls="-", label=f"{label_prefix}median")
`
and the root cause being inside matplotlib / numpy. 
Total stack is provided here :
[stack.txt](https://github.com/awslabs/gluon-ts/files/4244735/stack.txt)

Here i'm stuck, and i can't get any further to understand what i'm doing wrong or if it's a bug somewhere (as my usecase seems to be pretty simple and straightforward) or if i missed something obvious.

As attachment, the config of my environment, which might help.
[gluonts-requirements.txt](https://github.com/awslabs/gluon-ts/files/4244648/gluonts-requirements.txt)

Thanks for any help / answer / suggestion / other that anyone could provide.

Best regards
Nicholas
